modules/dev.js

```diff
@@ -0,0 +1,23 @@
+import { Employee } from "./employee.js";
+
+export class Dev extends Employee {
+    constructor(id, salary) {
+        super(id, salary);
+    }
+    
+    getJobType() {
+        return `I am a Dev`
```

Comment:
You need to create a private property for the text "I am a Dev" and return it a result of the function getJobType()